workers/generic-worker/multiuser_posix.go

```diff
@@ -25,6 +25,7 @@ func (task *TaskRun) formatCommand(index int) string {
 func platformFeatures() []Feature {
 	return []Feature{
 		&InteractiveFeature{},
+		&LoopbackVideoFeature{},
```

Comment:
We probably need to split platform features up into linux and darwin, since now they diverge. Does interactive work on mac? If not, we should remove it. Could be fun to try it out - maybe it is close enough to linux that it might work?

Alternative: we could see what is involved in getting a virtual video device running on macOS. Maybe there is something similar to v4l2loopback for mac? If so, would be nice to support both platforms.

Reply:
I've decided to _not_ split it up and instead just return a malformed payload in the case where a user tries to run this on a a mac.